tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "emitDecoratorMetadata": true,
     "extendedDiagnostics":true,
     "noErrorTruncation": true,
-    "sourceMap": true
+    "sourceMap": true,
   },
+  "exclude": ["./test"]
```

Comment:
if you add an exclude here, you have to put node_modules there too. otherwise it will try to compile them, as this is an override for the default exclude.
ref: https://www.typescriptlang.org/tsconfig#exclude

Reply:
No such thing is currently happening. Let's get together and check it out?

Reply:
restructured